.changes/2783-chat-list-ui-improvement.md

```diff
@@ -0,0 +1 @@
+- [Improvement] : Improve chat list design with better indication of mute, bookmark, notificiation count and other related data.
```

Comment:
I think we also want to indicate that we have a chat design playground in the settings, don't we ? That is good way to collect user feedback based on the chat design and improvement iterations on it :)

Reply:
it is dev and nightly only though. I'd not include it in the changelog.